### PR TITLE
General: update required node version to v16.13.2

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* Update to the most recent version of Node for all our tooling.

#### Jetpack product discussion

﻿* Internal reference: p3btAN-1D8-p2

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Is CI happy?
* Does everything build properly?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
 		"jetpack-js-tools": "workspace:*"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline package.json:81]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

Should we bump this to the 14.x release that shipped along side 16.13.2?

File: projects/github-actions/required-review/package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
 		"build": "ncc build src/main.js -o dist --source-map --license licenses.txt"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/github-actions/required-review/package.json:22]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/analytics/package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
 		"test": "NODE_ENV=test NODE_PATH=tests:. js-test-runner --jsdom 'glob:./test/*.@(jsx|js)'"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/analytics/package.json:23]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/api/package.json [hunk 4]
```diff
@@ -23,7 +23,7 @@
 		"test": "NODE_ENV=test NODE_PATH=tests:. js-test-runner --jsdom --initfile=test-main.jsx 'glob:./test/*.jsx'"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/api/package.json:26]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/babel-plugin-replace-textdomain/package.json [hunk 5]
```diff
@@ -25,7 +25,7 @@
 		"jest": "27.3.1"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/babel-plugin-replace-textdomain/package.json:28]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/base-styles/package.json [hunk 6]
```diff
@@ -25,7 +25,7 @@
 		"@wordpress/base-styles": "4.0.4"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/base-styles/package.json:28]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/components/package.json [hunk 7]
```diff
@@ -38,7 +38,7 @@
 		"test": "NODE_ENV=test NODE_PATH=tests:. js-test-runner --jsdom --initfile=test-main.jsx 'glob:./!(node_modules)/**/test/*.jsx'"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/components/package.json:41]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/config/package.json [hunk 8]
```diff
@@ -20,7 +20,7 @@
 		"jetpack-js-test-runner": "workspace:*"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/config/package.json:23]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/eslint-changed/package.json [hunk 9]
```diff
@@ -33,7 +33,7 @@
 		"eslint": ">=7.0.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/eslint-changed/package.json:36]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/eslint-config-target-es/package.json [hunk 10]
```diff
@@ -33,7 +33,7 @@
 		"eslint-plugin-es": "^4.1.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/eslint-config-target-es/package.json:36]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/i18n-check-webpack-plugin/package.json [hunk 11]
```diff
@@ -29,7 +29,7 @@
 		"webpack": "^5.0.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/i18n-check-webpack-plugin/package.json:32]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/i18n-loader-webpack-plugin/package.json [hunk 12]
```diff
@@ -30,7 +30,7 @@
 		"webpack": "^5.29.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/i18n-loader-webpack-plugin/package.json:33]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/idc/package.json [hunk 13]
```diff
@@ -41,7 +41,7 @@
 		"test": "NODE_ENV=test NODE_PATH=tests:. js-test-runner --jsdom --initfile=test-main.jsx 'glob:./!(node_modules)/**/test/*.jsx'"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/idc/package.json:44]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/partner-coupon/package.json [hunk 14]
```diff
@@ -31,7 +31,7 @@
 		"prop-types": "15.7.2"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/partner-coupon/package.json:34]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/remove-asset-webpack-plugin/package.json [hunk 15]
```diff
@@ -27,7 +27,7 @@
 		"webpack": "^5.8.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/remove-asset-webpack-plugin/package.json:30]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/storybook/package.json [hunk 16]
```diff
@@ -63,7 +63,7 @@
 		"webpack": "5.65.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/storybook/package.json:66]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/js-packages/webpack-config/package.json [hunk 17]
```diff
@@ -66,7 +66,7 @@
 		"./babel/preset": "./src/babel-preset.js"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/js-packages/webpack-config/package.json:69]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/admin-ui/package.json [hunk 18]
```diff
@@ -24,7 +24,7 @@
 	},
 	"devDependencies": {},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/admin-ui/package.json:27]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/connection-ui/package.json [hunk 19]
```diff
@@ -39,7 +39,7 @@
 		"webpack-cli": "4.9.1"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/connection-ui/package.json:42]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/identity-crisis/package.json [hunk 20]
```diff
@@ -33,7 +33,7 @@
 		"webpack": "5.65.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/identity-crisis/package.json:36]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/jitm/package.json [hunk 21]
```diff
@@ -24,15 +24,15 @@
...
+		"@automattic/jetpack-webpack-config": "workspace:^1.0.2-alpha",
 		"@wordpress/browserslist-config": "4.1.0",
 		"sass": "1.43.3",
 		"sass-loader": "12.4.0",
 		"webpack": "5.65.0",
 		"webpack-cli": "4.9.1"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/jitm/package.json:35]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/lazy-images/package.json [hunk 22]
```diff
@@ -25,13 +25,13 @@
...
 	"devDependencies": {
-		"@automattic/jetpack-webpack-config": "workspace:^1.0.1",
+		"@automattic/jetpack-webpack-config": "workspace:^1.0.2-alpha",
 		"copy-webpack-plugin": "10.2.0",
 		"intersection-observer": "0.12.0",
 		"webpack": "5.65.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/lazy-images/package.json:34]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/my-jetpack/package.json [hunk 23]
```diff
@@ -48,7 +48,7 @@
 		"webpack": "5.65.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/my-jetpack/package.json:51]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/packages/search/package.json [hunk 24]
```diff
@@ -34,15 +34,15 @@
 	},
 	"homepage": "https://github.com/Automattic/jetpack-search#readme",
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/packages/search/package.json:37]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/plugins/backup/package.json [hunk 25]
```diff
@@ -54,7 +54,7 @@
 		"webpack-cli": "4.9.1"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/plugins/backup/package.json:57]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/plugins/boost/package.json [hunk 26]
```diff
@@ -70,7 +70,7 @@
 	},
 	"homepage": "https://jetpack.com/boost/",
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/plugins/boost/package.json:73]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/plugins/boost/tests/e2e/package.json [hunk 27]
```diff
@@ -40,7 +40,7 @@
 		"jetpack-e2e-commons": "link:../../../../../tools/e2e-commons"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/plugins/boost/tests/e2e/package.json:43]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/plugins/jetpack/package.json [hunk 28]
```diff
@@ -203,7 +203,7 @@
 		"react-dom": "17.0.2"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/plugins/jetpack/package.json:206]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/plugins/jetpack/tests/e2e/package.json [hunk 29]
```diff
@@ -44,7 +44,7 @@
 		"jetpack-e2e-commons": "link:../../../../../tools/e2e-commons"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/plugins/jetpack/tests/e2e/package.json:47]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: projects/plugins/search/package.json [hunk 30]
```diff
@@ -22,7 +22,7 @@
 	},
 	"devDependencies": {},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline projects/plugins/search/package.json:25]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: tools/cli/package.json [hunk 31]
```diff
@@ -44,7 +44,7 @@
 		"yargs-parser": "20.2.4"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline tools/cli/package.json:47]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: tools/cli/skeletons/common/package.json [hunk 32]
```diff
@@ -24,7 +24,7 @@
 	},
 	"devDependencies": {},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline tools/cli/skeletons/common/package.json:27]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: tools/e2e-commons/package.json [hunk 33]
```diff
@@ -40,7 +40,7 @@
 		"yargs": "17.3.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline tools/e2e-commons/package.json:43]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: tools/js-test-runner/package.json [hunk 34]
```diff
@@ -36,7 +36,7 @@
 		"sinon-chai": "3.7.0"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline tools/js-test-runner/package.json:39]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

File: tools/js-tools/package.json [hunk 35]
```diff
@@ -45,7 +45,7 @@
 		"yaml": "1.10.2"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```
Comment (kraftbj) [inline tools/js-tools/package.json:48]:
```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

**Review by kraftbj — CHANGES_REQUESTED** [review]
The 14.x branch also had a security release shipped. Should we bump that as well? (all my inline comments are about that).

**Review by kraftbj — APPROVED** [review]

**Review by anomiex — APPROVED** [review]

**Comment by jeherve** [comment]
r238239-wpcom

**Pull request merged**